misc/docs-linter/urls.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"bufio"
```

Comment:
Please organize the imports

Reply:
Do you mainly mean separate the stdlibs & link imports? If I understood you correctly, I did this: [b722ecf](https://github.com/gnolang/gno/pull/2416/commits/b722ecf224d3fdb0efd9a3b464ba162282994af7)